internal/testdata/child_symbols.go

```diff
@@ -1,4 +1,3 @@
-// Package testdata
```

Comment:
Is this removed on purpose?

Reply:
Yes, there should only be one spot that has a comment for the package `testdata` otherwise it's weird for choosing the documentation hover string for the package.